### ISSUE TITLE
How to Compile it in Windows or open it qt creator 

### ISSUE THREAD

**Issue by 0x1330** [Vatsinator/vatsinator-legacy#3]
hi 

how i can open it in qt creator ? 

how to use cmake to compile for windows ?

thank ,

**Comment by garrappachc** [comment]
Hi,
QtCreator has support for CMake, so the only thing you need to have is CMake itself installed in your system. You can download it here: http://www.cmake.org/download/
As for version 0.1 only MinGW is supported, version 1.0 will have MS Visual Studio 2013 support.

**Comment by 0x1330** [comment]
what is this?

-- Could NOT find QUAZIP (missing:  QUAZIP_LIBRARIES QUAZIP_INCLUDE_DIR QUAZIP_ZLIB_INCLUDE_DIR QUAZIP_INCLUDE_DIRS)

**Comment by garrappachc** [comment]
You need to have QuaZIP dependency installed -> http://quazip.sourceforge.net/

**Comment by 0x1330** [comment]
how to install it ?

**Comment by garrappachc** [comment]
You need to download the source code, compile it (under QtCreator, for example) and install. Just like a typical Qt-based C++ library.
Btw, I am writing a page covering the subject of building Vatsinator under all supported platforms. All details will be described there. You just need to wait for a couple of days, until I finish it.

**Comment by 0x1330** [comment]
Thank You <3

**Comment by PaulWalkerUK** [comment]
I see the wiki has notes for building on Linux and this ticket has a few pointers for Windows - is there anything we should know before (a c++ novice) starts trying to build on Windows?

**Comment by garrappachc** [comment]
No, not really. I managed to build Vatsinator under QtCreator 4.1.0 with Qt 5.7.0 without any issues. The only thing I'd like to mention is that you should be on the develop branch, not master. Master is a little bit outdated.
